### PR TITLE
Feature: Add Command wrappers

### PR DESCRIPTION
**Pull request by deribaucourt**
Add configuration options to run bitabke through build tools like kas, docker crops, cqfd, custom...

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -88,18 +88,33 @@
         },
         "bitbake.pathToEnvScript": {
           "type": "string",
-          "default": "${workspaceFolder}/sources/poky/oe-init-build-env",
+          "examples": ["${workspaceFolder}/sources/poky/oe-init-build-env"],
```
Comment (idillon-sfl) [inline client/package.json:91]: I don't see any of the examples you set when navigating in vscode settings. Is it normal?

Comment (idillon-sfl) [reply client/package.json:91]: Screenshot from 2023-11-20 10-55-27

Comment (deribaucourt) [reply client/package.json:91]: image
They properly render if I ask VSCode for completions, using Ctrl+Space

Comment (deribaucourt) [reply client/package.json:91]: Thanks for your screenshot, it seems examples are not reported in the settings view, with no way to see them. Guess I'll have to add them to the description. I could open an issue on vscode for that...

Comment (deribaucourt) [reply client/package.json:91]: I pushed two commits to address this issue and clarify the doc.

**Review by idillon-sfl — APPROVED** [review]

**Pull request merged**